src/thundra-wrapper.js

```diff
@@ -100,19 +103,75 @@ class ThundraWrapper {
     async report(error, result, callback) {
         if (!this.reported) {
             this.reported = true;
-            const afterInvocationData = {
+
+            let afterInvocationData = {
                 error: error,
                 response: result
             };
+
+            const skipParseResponse = process.
+                env.thundra_lambda_skip_parse_response === 'true' ? true : false;
```

Comment:
Since this check is HTTP response specific, I think we can use `thundra_lambda_http_responseCheck_skip` as environment variable. 

The reason why I have used this naming convention is that 
- Later there might be other environment variables related with HTTP domain in `thundra_lambda_http_...` format
- Later there might be other environment variables related with HTTP response check (not just about skipping but customizing check behaviour) in `thundra_lambda_http_responseCheck...` format

Reply:
👍